modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -68,10 +178,34 @@ $ oc create -f bmh.yaml
 .Example output
 [source,terminal]
 ----
-secret/worker1-bmc created
-baremetalhost.metal3.io/worker1 created
+secret/openshift-worker-<num>-network-config-secret created
+secret/openshift-worker-<num>-bmc-secret created
+baremetalhost.metal3.io/openshift-worker-<num> created
 ----
++
+Where `<num>` is the worker number.
 
+. Power on and inspect the bare-metal node:
```

Comment:
The node being added should be in the power down state before running the step to create the bmh above. The node will be powered on when the create command is executed. It will move from states registering, then inspecting, provisioning and finally provisioned. if all goes well The progress of the states can be checked with running the below get bmh command in  regular intervals.

Reply:
Updated As Suggested.